db/data/regions/AU/en.yml

```diff
@@ -8,29 +8,36 @@ en:
             default: Suburb
             optional: Suburb (optional)
           errors:
-            blank: Enter a suburb
-            too_long: The suburb name is too long (maximum is 255 characters)
+            blank_instructional: Enter a suburb
+            blank_informative: Suburb is missing
+            too_long_instructional: The suburb name is too long (maximum is 255 characters)
+            too_long_informative: Suburb is too long
             contains_emojis: Suburb cannot contain emojis
```

Comment:
`cannot` should be `can't` in all instances

Reply:
Not modifying existing ones - just adding our new ones.